blueprint/core/blueprint_ReactApplicationRoot.h

```diff
@@ -44,8 +95,18 @@ namespace blueprint
             // Install React.js backend rendering methods
             registerNativeRenderingHooks();
 
-            // And install view types
-            installNativeViewTypes();
+            // Initialise the ViewManager
+            initViewManager();
+
+#if JUCE_DEBUG
+            //TODO: Keep error handler here in JUCE_DEBUG builds only or should
```

Comment:
Good question. I'd say let's always install this handler, because it's always up to the user if they want to override it in their release builds or something

Reply:
Yeah sounds like a good plan.